### PR TITLE
Add tests for Math max and min

### PR DESCRIPTION
**Pull request by dhong44**
Add tests for Math max and min.

Signed-off-by: Daniel Hong <daniel.hong@live.com>

### CODE REVIEW

**Comment by dhong44** [comment]
@fjeremic I've updated the tests as you recommended in https://github.com/eclipse/openj9/pull/2973#pullrequestreview-158142608. Have also confirmed that these tests do expose corner cases such as the one patched in https://github.com/eclipse/omr/pull/2967#issuecomment-421469732. This is now ready for review.

File: test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Math.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 import org.testng.annotations.Test;
 import org.testng.Assert;
 
-@Test(groups = { "level.sanity" })
+@Test(groups = { "level.sanity" }, invocationCount = 10)
```
[the remark this answers is not in the document]
Comment (dhong44) [reply test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Math.java]: 2e5aa80

File: test/functional/Java8andUp/playlist.xml [hunk 2]
```diff
@@ -1025,6 +1025,32 @@
 		</groups>
 	</test>
 
+	<test>
+		<testCaseName>JCL_TEST_MathMethods</testCaseName>
+		<variations>
+			<variation>-Xjit:count=1,disableAsyncCompilation</variation>
```
[the remark this answers is not in the document]
Comment (dhong44) [reply test/functional/Java8andUp/playlist.xml]: a7cc1a9

File: test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Math.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
+package org.openj9.test.java.lang;
```
[the remark this answers is not in the document]
Comment (dhong44) [reply test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Math.java]: The tests in this folder did not follow a consistent formatting, in 088b765 all have been changed to follow the formatting in https://github.com/eclipse/openj9/blob/088b765f32c38597096ad0f626e099cd865edeaf/test/functional/Java11andUp/src/org/openj9/test/nestmate/NestmateTest.java#L1-L29

File: test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_RuntimePermission.java [hunk 4]
```diff
@@ -21,6 +21,13 @@
  *******************************************************************************/
 package org.openj9.test.java.lang;
 
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeMethod;
+import org.testng.AssertJUnit;
+
+package org.openj9.test.java.lang;
```
Comment (fjeremic) [inline test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_RuntimePermission.java]: @dhong44 some test failures because of this. The `package` is already defined. Looks like a copy/paste error. Can you make the appropriate fixes and I'll relaunch the builds.

Comment (dhong44) [reply test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_RuntimePermission.java]: 1a2c592

**Comment by fjeremic** [comment]
Jenkins test sanity xlinux,zlinux JDK8,JDK11

**Comment by AdamBrousseau** [comment]
Jenkins line endings check

**Comment by fjeremic** [comment]
@dhong44 some issues with the builds:

```
15:59:56 if [ $? -eq 0 ] ; then echo ""; echo "JCL_TEST_MathMethods_0""_PASSED"; echo ""; else echo ""; echo "JCL_TEST_MathMethods_0""_FAILED"; echo ""; fi; } 2>&1 | tee -a "/home/jenkins/jenkins-agent/workspace/PullRequest-Sanity-JDK8-linux_390-64_cmprssptrs-OpenJ9/openj9/test/TestConfig/scripts/testKitGen/../../../TestConfig/test_output_15379904224680/TestTargetResult";
15:59:56 /bin/sh: 1: Syntax error: "(" unexpected (expecting "}")
15:59:56 autoGen.mk:1020: recipe for target 'JCL_TEST_MathMethods_0' failed
```

Can you please take a look?

**Comment by fjeremic** [comment]
@dhong44 can we use single quotes to avoid having to escape anything (now and in the future). Here is an example:
https://github.com/eclipse/openj9/pull/2665/files#diff-be4413663bce2480d3a88638578a34b8R77

**Comment by dhong44** [comment]
I've changed the command line options to use single quotes. Since the changes have already 
been approved, I've done a squash and rebase as well.

**Comment by fjeremic** [comment]
Jenkins test sanity xlinux,zlinux JDK8,JDK11
